src/pages/declaration-consent/declaration-consent.ts

```diff
@@ -78,11 +80,13 @@ export class DeclarationConsentPage {
       .runAuthentication('Please authenticate yourself to proceed')
       .then((isAuthenticated: boolean) => {
         if (isAuthenticated) {
+          this.faultStore.setDebriefConsentStatus(this.checkDebrief);
```

Comment:
These aren't really faults, should we store them here?

Reply:
sausages